source/src_pw/potential.h

```diff
@@ -1,94 +1,91 @@
 #ifndef POTENTIAL_H
 #define POTENTIAL_H
 
+#include "../module_base/complexmatrix.h"
 #include "../module_base/global_function.h"
 #include "../module_base/global_variable.h"
 #include "../module_base/matrix.h"
-#include "../module_base/complexmatrix.h"
-#include "module_esolver/esolver_ks_lcao.h"
 #include "../module_pw/pw_basis.h"
+#include "module_esolver/esolver_ks_lcao.h"
+#include "module_esolver/esolver_ks_lcao_tddft.h"
```

Comment:
There is no need to add these two headers here?

Reply:
Yes, I will delete them.

Reply:
I delete friend class solver_ks_lcao and move set_vrs_tddft from private to public now.